EIPS/eip-4955.md

```diff
@@ -0,0 +1,85 @@
+---
+eip: 4955
+title: Non-Fungible Token Metadata Standard Extension
+description:  A metadata standard extension schema for non-fungible tokens
+author: Ignacio Mazzara (@nachomazzara)
+discussions-to: https://ethereum-magicians.org/t/eip-4955-non-fungible-token-metadata-namespaces-extension/8746
+type: Standards Track
+category: ERC
+status: Draft
+requires: 721, 1155
+created: 2022-03-29
+---
+
+## Abstract
+
+The following standard allows for the implementation of a standard schema for NFTs metadata.
+
+## Motivation
+
+A standard schema allows wallets, marketplaces, metaverses, etc. to work with any NFT.
+
+This standard is inspired by the projects like marketplaces and metaverses which want to leverage the usage of NFTs by rendering them using custom 3d representations or any other new attributes.
```

Comment:
I think you can definitely expand on this a bit. Are projects already doing this, but haphazardly?